{
    "data_folder_path": "data/CVE-2014-8991",
    "pull_requests_urls": [
        "https://github.com/pypa/pip/pull/2122"
    ],
    "cve_name": "CVE-2014-8991",
    "repo": "pypa/pip",
    "warnings": {
        "cve": "CVE-2014-8991",
        "warnings": [
            {
                "type": "NOT_MODIFIED",
                "file_name": "pip/utils/build.py",
                "message": "not a modified file (status: FileStatus.ADDED)\n"
            }
        ]
    },
    "files_extension_stats": {
        ".txt": 1,
        ".py": 6
    },
    "file_number": 7,
    "removed_files_number": 0,
    "added_files_number": 0,
    "nb_lines_affected": 442,
    "private_key_files": [],
    "languages": [
        "Python"
    ]
}